gobblin-utility/src/test/java/gobblin/util/binpacking/WorstFitDecreasingBinPackingTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.util.binpacking;
+
+import java.util.List;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.beust.jcommander.internal.Lists;
+
+import gobblin.source.workunit.Extract;
+import gobblin.source.workunit.MultiWorkUnit;
+import gobblin.source.workunit.WorkUnit;
+
+
+public class WorstFitDecreasingBinPackingTest {
+
+  public static final String WEIGHT = "weight";
+
+  @Test
+  public void testPack()
```

Comment:
As a general note it is better to break down a test class into logical test methods. The idea is to have as few Asserts in a test method as possible. This is makes it easy to debug when a test fails.